.github/workflows/test.yml

```diff
@@ -6,9 +6,9 @@ on:
       - staging
   workflow_dispatch:
       inputs:
-        name:
+        stage:
           description: 'Stage'
-          required: true
+          required: false
           default: 'production'
   schedule:
    - cron:  '43 3 * * *' # daily
```

Comment:
Any reason for 0343 in particular?
Should we run this more than once a day?

Reply:
0343 was arbitrary, and chose just so it wasn't midnight as to compete with other processes that just default to that.

We should consider running this hourly once we get everything set up and when we're actually monitoring it or having it push failure notifications to slack (or whatever)